src/components/app.tsx

```diff
@@ -111,10 +112,11 @@ const viewMap = makeViewMap();
 
 export default function App(props: AppProps): JSX.Element {
   const classes = useStyles();
-  const { transportFactory, trajectoryManagerFactory } = props.appConfig;
+  const { transportFactory, trajectoryManagerFactory, appResources } = props.appConfig;
   const [transport, setTransport] = React.useState<RomiCore.Transport | undefined>(undefined);
   const [buildingMap, setBuildingMap] = React.useState<RomiCore.BuildingMap | undefined>(undefined);
   const trajManager = React.useRef<RobotTrajectoryManager | undefined>(undefined);
+  const ResourceManager = React.useRef<ResourceConfigurationsType | undefined>(undefined);
```

Comment:
I don't think this should be PascalCase, `ResourceConfigurationsType` is a `Record<string, SubResourceConfigurationsType>`, usually we should only use `PascalCase` for newable objects or special cases like react functional components.

Reply:
Sorry, you're right, that's a leftover from the refactor. Thanks!

Reply:
Just to check, you're talking me about `const ResourceManager` -> `const resourceManager`, right?

Because the type `ResourceConfigurationsType` should be in PascalCase accordingly to https://basarat.gitbook.io/typescript/styleguide#type. Example: https://basarat.gitbook.io/typescript/styleguide#type-vs-interface

Reply:
yep